src/app/sponsors/sponsors.component.ts

```diff
@@ -27,8 +29,16 @@ import { SponsorsService } from './sponsors.service';
 export class SponsorsComponent implements OnInit {
 
   public sponsors: Sponsor[];
+  public partners: Sponsor[];
+  public showSecondRow = false;
 
-  constructor(private sponsorsService: SponsorsService) { }
+  constructor(private sponsorsService: SponsorsService, private location: Location, private router: Router) {
+    router.events.subscribe((event) => {
```

Comment:
Any particular reason it's in the constructor instead on OnInit?

Reply:
I was just going off of examples online.